.eslintrc.cjs

```diff
@@ -3,29 +3,23 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
-    jest: true
+    jest: true,
```

Comment:
Do we still need this line if we're using Vitest?

Reply:
Hmm... Maybe for the globals. Let's see ! :goes to console:

Reply:
Switched to the vitest env